unicodetools/src/main/java/org/unicode/text/UCD/MakeUnicodeFiles.java

```diff
@@ -53,7 +59,64 @@ public class MakeUnicodeFiles {
     static boolean DEBUG = false;
 
     public static void main(String[] args) throws IOException {
+
+        boolean clean = Arrays.asList(args).contains("-c");
+
+        if (clean) {
+
+            // Remove the bin file so that changes to the dev directory aren't ignored
+            // The index unicode properties (eg Alphabetic.bin) don't need to be removed
+            // because they are rebuilt if the source files change
+
+            File binFile =
+                    new File(
+                            Settings.Output.GEN_DIR + "BIN",
+                            "UCD_Data" + Settings.latestVersion + ".bin");
+            binFile.delete();
+
+            // remove the old files in the output directory
+
+            Files.walk(Path.of(Settings.Output.GEN_DIR + "UCD"))
+                    .sorted(Comparator.reverseOrder())
+                    .map(Path::toFile)
+                    .forEach(File::delete);
+        }
+
         generateFile();
+
+        if (clean) {
```

Comment:
I find it surprising/confusing that the "clean" option should also drive copying the files back to the input.

It would be better to make another new option like "--changesToInput" .

Reply:
ping -- I really think it would be better to make a second option, with another readable option name (not just renaming the Java variable)

Reply:
I don't disagree with @markusicu but from the standpoint of practicality, do we see ourselves or anyone else using one option without the other?